Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -4711,17 +4710,34 @@ void CGame::RegisterClientTriggeredEventUsage(CPlayer* pPlayer)
     if (!pPlayer || !pPlayer->IsPlayer() || pPlayer->IsBeingDeleted())
         return;
 
-    m_mapClientTriggeredEvents[pPlayer]++;
+    int now = GetTickCount64_();
```

Comment:
GetTickCount64_ returns `long long` which is 8 bytes long. But `int` is only 4 bytes long. That means that when your timer will cover INT_MAX your logic gets broken.

Reply:
For example: 
```
struct 
{
    long long m_llTicks;
    uint32_t m_uiCounter;
};
```

Reply:
Resolved